src/test/java/uk/gov/hmcts/ccd/endpoint/std/CaseAccessEndpointIT.java

```diff
@@ -60,7 +60,7 @@
     private JdbcTemplate template;
 
     @MockBean
-    @Qualifier(AMCaseUserRepository.QUALIFIER)
+    @Qualifier(AMCaseUserRepository.ACCESS_MANAGEMENT_QUALIFIER)
```

Comment:
Same here - why going away from the convention here?

Reply:
Reverted to AMCaseUserRepository.QUALIFIER.